projects/common/src/lib/layer-collections/layer-collection-list/layer-collection-list.component.ts

```diff
@@ -211,7 +233,7 @@ class LayerCollectionItemDataSource extends DataSource<LayerCollectionItemDict>
         const offset = this.offset;
         const limit = this.scrollFetchSize;
 
-        return this.getCollectionItems(offset, limit).pipe(
+        return from(this.getCollectionItems(offset, limit)).pipe(
```

Comment:
because `getCollectionItems` now uses the new layersService built upon the openapi-client and return a Promise and not an observable anymore. In order to keep the rest of the logic I convert the promise to an observable here.